packages/user-interface/src/pages/MessagePage.tsx

```diff
@@ -1,59 +1,77 @@
-import { useGetTakenV2Query, TaakV2 } from "@nl-portal/nl-portal-api";
+import {
+  Bericht,
+  TaakV2,
+  useGetBerichtQuery,
+  useGetTakenV2Query,
+} from "@nl-portal/nl-portal-api";
 import { Paragraph } from "@gemeente-denhaag/typography";
 import { FormattedMessage } from "react-intl";
 import { useOutletContext, useParams } from "react-router-dom";
 import BackLink from "../components/BackLink";
 import PageGrid from "../components/PageGrid";
 import PageHeader from "../components/PageHeader";
 import TasksList from "../components/TasksList";
-import { MessageType } from "../components/Message";
-import { messages } from "./MessagesPage";
 import { RouterOutletContext } from "../contexts/RouterOutletContext";
+import styles from "./MessagePage.module.scss";
+import Skeleton from "../components/Skeleton.tsx";
 
 const MessagePage = () => {
   const { id } = useParams();
   const { paths } = useOutletContext<RouterOutletContext>();
-  const {
-    data: tasksData,
-    loading: taskLoading,
-    error: tasksError,
-  } = useGetTakenV2Query({
+  const { data: tasksData, loading: taskLoading } = useGetTakenV2Query({
     variables: { zaakId: id },
   });
-  const loading = taskLoading;
-  const error = tasksError;
-  const message: MessageType = messages.find((m) => m.id === Number(id))!;
-  const tasks = tasksData?.getTakenV2.content as TaakV2[] | undefined;
+  const {
+    data: messageData,
+    loading: messageLoading,
+    error: messageError,
+  } = useGetBerichtQuery({
+    variables: { id: id },
+  });
+  const tasks: TaakV2[] | undefined = tasksData?.getTakenV2.content as
+    | TaakV2[]
+    | undefined;
+  const message: Bericht | undefined = messageData?.getBericht as
+    | Bericht
+    | undefined;
+
+  if (messageLoading) {
+    return (
+      <div>
+        <Skeleton />
+      </div>
+    );
+  }
 
-  if (error) {
-    <div>
-      <Paragraph>
-        <FormattedMessage id="messagePage.fetchError" />
-      </Paragraph>
-    </div>;
+  if (messageError || !message) {
+    return (
+      <div>
+        <BackLink routePath={paths.messages} />
+        <Paragraph>
+          <FormattedMessage id="messagePage.fetchError" />
+        </Paragraph>
+      </div>
+    );
   }
 
   return (
     <PageGrid>
       <div>
         <BackLink routePath={paths.messages} />
-        <PageHeader loading={loading} title={!loading && message.titel} />
+        <PageHeader
+          loading={messageLoading}
+          title={!messageLoading && message?.onderwerp}
+        />
       </div>
       <TasksList
-        loading={loading}
-        showEmpty={false}
-        titleTranslationId={null}
-        tasks={tasks}
-      />
-      <section>
-        <Paragraph>{message.bericht}</Paragraph>
-      </section>
-      <TasksList
-        loading={loading}
+        loading={taskLoading}
         showEmpty={false}
         titleTranslationId={null}
         tasks={tasks}
       />
+      <div className={styles["message__content-text"]}>
```

Comment:
Please make a separate component for this. So we can also handle loading and error states and we can prevent styling on page lvl. There is also a new design for the hackathon where there are no tasks for now. Only documents. Do you have access to that design?